berkeley-function-call-leaderboard/bfcl/model_handler/gpt_handler.py

```diff
@@ -19,9 +19,10 @@ class OpenAIHandler(BaseHandler):
     def __init__(self, model_name, temperature=0.001, top_p=1, max_tokens=1000) -> None:
         super().__init__(model_name, temperature, top_p, max_tokens)
         self.model_style = ModelStyle.OpenAI
-        self.client = OpenAI(api_key=os.getenv("OPENAI_API_KEY"))
 
     def inference(self, prompt, functions, test_category):
+        # Move it here so eval checker can run w/o instantiating API key since we don't use it at that point
```

Comment:
This is not necessary. `eval_checker` can run without the OpenAI API keys. If `OPENAI_API_KEY` is not supplied, it would just be `None` and won't break the evaluation pipeline.